### PR TITLE
Added a light for RGB and Temperature

### PR DESCRIPTION
**Pull request by merlinschumacher**
Hi,

I've added a light that can receive a color temperature or an RGB value. This is useful for some modern Zigbee lamps.

### CODE REVIEW

File: devices/light.js [hunk 1]
```diff
@@ -1292,4 +1292,295 @@ module.exports = function(RED) {
...
+                node.status({fill: "red", shape: "dot", text: "OFF"});
+            }
+        }
+
+        /******************************************************************************************************************
+         * called when state is updated from Google Assistant
+         *
+         */
+        this.updated = function(device) {   // this must be defined before the call to clientConn.register()
+            let states = device.states;
```
Comment (Chrischi-) [inline devices/light.js:1393]: let command = device.command;

File: devices/light.js [hunk 2]
```diff
@@ -1292,4 +1292,295 @@ module.exports = function(RED) {
...
+            RED.log.debug("LightRgbTempNode(updated): states = " + JSON.stringify(states));
+
+            node.updateStatusIcon(states);
+            
+            let msg = {
+                topic: node.topicOut,
+                device_name: device.properties.name.name,
+                payload: {
```
Comment (Chrischi-) [inline devices/light.js:1401]: command: command,

File: devices/light.js [hunk 3]
```diff
@@ -1292,4 +1292,295 @@ module.exports = function(RED) {
...
+         */
+        this.updated = function(device) {   // this must be defined before the call to clientConn.register()
+            let states = device.states;
+            RED.log.debug("LightRgbTempNode(updated): states = " + JSON.stringify(states));
+
+            node.updateStatusIcon(states);
+            
+            let msg = {
+                topic: node.topicOut,
+                device_name: device.properties.name.name,
```
Comment (Chrischi-) [inline devices/light.js:1400]: command: command,

File: devices/light.js (hunk 1 above)
Comment (Chrischi-) [reply devices/light.js:1393]: to use the current code base.

File: devices/light.js (hunk 3 above)
Comment (Chrischi-) [reply devices/light.js:1400]: to use the current code base.

**Review by Chrischi- — CHANGES_REQUESTED** [review]
to use the current code base.

File: devices/light.js (hunk 3 above)
Comment (merlinschumacher) [reply devices/light.js:1401]: corrected with #cb28310

File: devices/light.js (hunk 1 above)
Comment (merlinschumacher) [reply devices/light.js:1393]: corrected with #cb28310

**Pull request merged**